model/src/w3nmlboundmd.F90

```diff
@@ -0,0 +1,392 @@
+#include "w3macros.h" 
+!/ ------------------------------------------------------------------- /
+      MODULE W3NMLBOUNDMD
+!/
+!/                  +-----------------------------------+
+!/                  | WAVEWATCH III           NOAA/NCEP |
+!/                  |           M. Accensi              |
+!/                  |                                   |
+!/                  |                        FORTRAN 90 |
+!/                  | Last update :         27-May-2021 |
+!/                  +-----------------------------------+
+!/
+!/    For updates see subroutines.
+!/
+!  1. Purpose :
+!
+!     Manages namelists from configuration file ww3_bound.nml for ww3_bound program
+!
+!/ ------------------------------------------------------------------- /
+
+  ! module defaults
+  IMPLICIT NONE
+
+  PUBLIC
+
+  ! bound structure
+  TYPE NML_BOUND_T
```

Comment:
When we add in the doxygen, we'll need variable descriptions for all of these variables in the routines.  Maybe we should just add these now? (even if not in doxygen format, just give quick descriptions of the variables).